cmd/start.go

```diff
@@ -266,7 +266,7 @@ func runStartServer(cmd *cobra.Command, args []string) error {
 			// If desired mode is prod, then server initialization is a manual next step to be taken by the user.
```

Comment:
change the "prod" in the comment.
In general, please also search for any occurrences of "prod" (we often used it in place of "production")